src/cargo/core/resolver/errors.rs

```diff
@@ -362,12 +362,23 @@ pub(super) fn activation_error(
                         let _ = writeln!(&mut msg, "  version {} is not cached", summary.version());
                     }
                     IndexSummary::Unsupported(summary, schema_version) => {
-                        let _ = writeln!(
+                        if let Some(rust_version) = summary.rust_version() {
+                            // HACK: technically its unsupported and we shouldn't make assumptions
```

Comment:
What do you mean by `technically its unsupported`?

Reply:
For now, the `Unsupported` branch means that we can parse the entry but the schema version was bumped. How we interpret the fields could have changed.

It'll be even worse in a follow up where I'll add "failed to parse but could get basic data".

Reply:
Got it. Thank you!